src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -47,20 +47,13 @@
             (update-in [:wallet :ui :send] dissoc :route)
             (update-in [:wallet :ui :send] dissoc :loading-suggested-routes?))}))
 
-(rf/reg-event-fx :wallet/select-send-account-address
- (fn [{:keys [db]} [{:keys [address stack-id]}]]
-   {:db (-> db
-            (assoc-in [:wallet :ui :send :send-account-address] address)
-            (update-in [:wallet :ui :send] dissoc :to-address))
-    :fx [[:dispatch [:navigate-to-within-stack [:wallet-select-asset stack-id]]]]}))
-
 (rf/reg-event-fx :wallet/clean-send-address
  (fn [{:keys [db]}]
    {:db (update-in db [:wallet :ui :send] dissoc :recipient :to-address)}))
 
 (rf/reg-event-fx
  :wallet/select-send-address
- (fn [{:keys [db]} [{:keys [address token recipient stack-id]}]]
```

Comment:
a minor detail to this api to make it more readable, previously it looked as though we are using all of the token details in this endpoint